.github/workflows/build-package.yml

```diff
@@ -53,7 +53,8 @@ jobs:
           --secret_key "${PACKAGING_SECRET_KEY}" \
           --passphrase "${PACKAGING_PASSPHRASE}" \
           --output_dir "$(pwd)/packages/" \
-          --input_files_dir "$(pwd)"
+          --input_files_dir "$(pwd)" \
+          --output_validation
```

Comment:
Enables auto validation which will check the ci outputs and fails the pipeline if there is a warning other than packaging-ignore.yml